TextField/TextField.jsx

```diff
@@ -199,6 +200,10 @@ class TextField extends Component {
       props.id = 'tf-' + this.id;
     }
 
+    const intermediateStyle = {
```

Comment:
delete this

Reply:
That are the defaults, I would like to keep them. You can't see them from the outside, but if you are really sure, I can remove them